lib/domain/usecase/room/create_new_group_chat_interactor.dart

```diff
@@ -31,6 +31,11 @@ class CreateNewGroupChatInteractor {
       );
 
       if (roomId.isNotEmpty) {
+        if (createNewGroupChatRequest.enableEncryption == true) {
+          await matrixClient
+              .getRoomById(roomId)
+              ?.setHistoryVisibility(HistoryVisibility.joined);
+        }
```

Comment:
IMO, we can add this state to `initialState` of `matrixClient.createGroupChat` method.

Reply:
done